fuzzers/symcc_aflplusplus/builder.Dockerfile

```diff
@@ -35,7 +35,14 @@ RUN cd /afl && unset CFLAGS && unset CXXFLAGS && \
     cp utils/aflpp_driver/libAFLDriver.a /
 
 # Install the packages we need.
-RUN apt-get install -y ninja-build flex bison python zlib1g-dev cargo 
+RUN apt-get install -y ninja-build flex bison python zlib1g-dev
```

Comment:
Do an apt-get update first or there can  be issues.

Reply:
Done.